src/core/runtime/src/enable-trusted-access-for-services-step.ts

```diff
@@ -86,9 +86,6 @@ export const handler = async (input: EnableTrustedAccessForServicesInput) => {
   console.log('Security account registered as delegated administrator for Guard Duty in the organization.');
 
   // Get all the parameter history versions from SSM parameter store
-  const parameterHistoryList = await ssm.getParameterHistory('/accelerator/version');
-  // Finding the first entry of the parameter version
-  const installerVersion = parameterHistoryList.find(p => p.Version === 1);
-  const installerVersionValue = JSON.parse(installerVersion!.Value!);
-  return !installerVersionValue.AcceleratorVersion ? '<1.2.2' : installerVersionValue.AcceleratorVersion;
+  const installedVersionParam = await ssm.getParameter('/accelerator/installed-version');
+  return installedVersionParam.Parameter?.Value || '<1.2.2';
```

Comment:
Should we not throw an error (and not assume <1.2.2) if we get here?  otherwise customers with 1.2.2,1.2.3,1.2.4 are going to be tagged as <122.  i.e. should the parameter not always exist and be properly populated?